### PR TITLE
Create 2019-01-07--go-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by eingenito**

### CODE REVIEW

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    1. Blockers/asks (no updates)
+    2. Announcements
+    3. Agenda
+  - Async:
+    - Every large endeavor gets a meta issue ("epic issue").
+    - Updates are posted in these issues.
+- Gateway Issue - @stebalien
+  - Believed to be https://github.com/ipfs/go-bitswap/issues/53
+    - We're not connecting to providers.
+    - Deeper issue: we should be reconnecting to peers in a session.
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: Note to reader: there are more complex issues than the ones that can be solved by a hotfix. The "Fix the Gateways" meeting was productive, Now we have some really good ideas anda tiger team / task force spearheaded by @eefahy

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+- CORs - @stebalien
+  - Our CORs headers are a bit of a mess.
+  - Separate implementations for the gateway/api
+  - Some fixes, not all: https://github.com/ipfs/go-ipfs/pull/5893
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: The Gateway doesn't need CORS in the first place. clients only issue GET requests to it. Going to post more comments on the PR.

Also, as communicated in the call, I volunteer to tag team this with you @Stebalien. I now see that @lidel has also been on top of it and provided lots of inputs https://github.com/ipfs/go-ipfs/pull/5893.

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    - More catchup, review, maintainer stuff.
+    - Make sure the bitswap bug affecting the gateway gets fixed.
+    - Cut a release.
+   
+
+@bigs
+  - Done: 
+    - vacation!
+    - email catch up
+    - unblocking daemon progress
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: @bigs could you be more descriptive here? What does it mean "unblocking daemon progress"? Is there any trail of breadcrumbs to issues and PRs?

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  - Blocked:
+    - Not really, but would really appreciate feedback on the providing PR
+  - Next:
+    - Work out GC/Providing issue with new providing tracking mechanism
+    - There's still a few small things to tie up with this new providing mechanism, so continuing work on them
+
+@momack2
+ - Done:
+   - Holidays
+   - Local/Offline Talk from London Meetup published to Youtube! (writing up more on that - next monthly is Jan 23) - https://twitter.com/momack28/status/1082350145994289153
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: Wowowowowow!!! \o/ :D Nice! 👏🏽👏🏽👏🏽

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md (hunk 2 above)
Comment (lidel) [reply meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: > The Gateway doesn't need CORS in the first place. 

I think that is not true: CORS on Gateway  is useful for API subset exposed there. Websites are able to read data via cross-origin XHR to https://ipfs.io/api/v0/dns/ipfs.io?r=true and AFAIK that is why our Gateway returns `Access-Control-Allow-Origin: *`

File: meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    - found a DoS vulnerability in the current QUIC specification and proposed a solution
+    - started interop testing with other draft-17 QUIC implementations. Still *a lot* of work to do, but this already proved hugely benefitial in finding issues
+    - fixed tons of issues in quic-go
+  - Blocked: ---
+  - Next: focussing on interop testing for now
+
+@Magik6k:
+ - Done:
+    - A Bunch of reviews
+    - Ran go-ipfs in web browser - https://github.com/ipfs/go-ipfs/pull/5878 (WIP)
```
Comment (lidel) [inline meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md]: Wow! Its not just web browser, I imagine WASM runtime will be quite popular in IoT, eventually :)

**Pull request merged**